api.rest

```diff
@@ -4,4 +4,13 @@ http://localhost:8080
 --
 GET /health
 
-
+--
+POST /invite_user
+{
+    "name": "mareq",
+    "email": "marek@repositive.io",
+    "organisation_id": "Repositive",
```

Comment:
I had a play around, and the `InviteUserToOrgCmd` looks like this in my branch:

```rust
#[derive(Debug, Clone, Serialize, Deserialize)]
pub struct InviteUserToOrgCmd {
    email: String,
    membership_role: MembershipRole,
    name: String,
    organisation_id: OrganisationId,
}
```

Because `organisation_id` is an `OrganisationId` (`== Uuid`), this _must_ be a UUID string or the request will fail to parse.